doc/source/user/quickstart.rst

```diff
@@ -348,6 +347,7 @@ existing array rather than create a new one.
 
 ::
 
+    >>> rg = np.random.default_rng(1)
```

Comment:
Perhaps a comment could be added here as well, to briefly describe what this line does without distracting from the overall theme of the example (illustrating in-place operations)

Reply:
Makes sense I'll add a comment (formatted like others on the page) - "create instance of default random number generator"